packages/block-editor/src/components/spacing-sizes-control/spacing-input-control.js

```diff
@@ -0,0 +1,240 @@
+/**
```

Comment:
I think we can probably split this into some smaller sub-components to simplify it a little, but we can circle back on that in a future PR I think